go.sum

```diff
@@ -36,6 +36,8 @@ cloud.google.com/go/storage v1.10.0/go.mod h1:FLPqc6j+Ki4BU591ie1oL6qBQGu2Bl/tZ9
 code.cloudfoundry.org/bytefmt v0.0.0-20180906201452-2aa6f33b730c h1:VzwteSWGbW9mxXTEkH+kpnao5jbgLynw3hq742juQh8=
 code.cloudfoundry.org/bytefmt v0.0.0-20180906201452-2aa6f33b730c/go.mod h1:wN/zk7mhREp/oviagqUXY3EwuHhWyOvAdsn5Y4CzOrc=
 dmitri.shuralyov.com/gpu/mtl v0.0.0-20190408044501-666a987793e9/go.mod h1:H6x//7gZCb22OMCxBHrMx7a5I7Hp++hsVxbQ4BYO7hU=
+github.com/ALTree/bigfloat v0.0.0-20201218142103-4a33235224ec h1:tPYYUIlnk8STUTx7TEcZqCgk0zvxL6KH6eDSTa38zBI=
+github.com/ALTree/bigfloat v0.0.0-20201218142103-4a33235224ec/go.mod h1:+NaH2gLeY6RPBPPQf4aRotPPStg+eXc8f9ZaE4vRfD4=
```

Comment:
where is this used? we shouldn't rely on external floating point imlementations since it may cause different values in different implementations and may affect consensus

Reply:
It's used in https://github.com/spacemeshos/go-spacemesh/blob/tortoise-ticker/tortoisebeacon/tortoise_beacon.go#L808
I use it because I need to calculate 2 to the power of `big.Float`, and the `Pow` method doesn't exist on https://golang.org/pkg/math/big/#Float